ext/RMagick/rmimage.c

```diff
@@ -9061,6 +9061,9 @@ Image_monitor_eq(VALUE self, VALUE monitor)
         (void) SetImageProgressMonitor(image, rm_progress_monitor, (void *)monitor);
     }
 
+#if defined(_WIN32)
+    rb_warn("Image#monitor= does not work on Windows");
+#endif
```

Comment:
Can we also pull this commit onto a separate PR? I've set up the commit settings to only allow squashing PRs, so it would be good to create separate PRs for different changes.

Reply:
🆗

Reply:
https://github.com/rmagick/rmagick/pull/344